R/get-results.r

```diff
@@ -47,7 +47,8 @@ id_scenarios <- function(directory){
         x <- unlist(strsplit(all.dirs[i], split="/"))
         return(x[length(x)])
     })
-    scenarios <- temp.dirs[grepl("[A-Z0-9-]+-[a-z]+",temp.dirs)]
+    scenarios <- temp.dirs[grepl("[A-Z0-9-]{2}-[a-z]+",temp.dirs)]
```

Comment:
I'm not sure I see how this change helps things. E.g.

```
> grepl("[A-Z0-9-]{2}-[a-z]+", "SS3-test")
[1] TRUE
```

Even if it did only find directories with one letter and one number followed by dash and lowercase letters, that wouldn't match typical scenario folders, would it? (e.g. `D0-M1-F0-cod.txt`).

Maybe this is a case where `user_scenarios` needs to be specified? We can't guess all possible ways scenario folders might look and probably better to be liberal with what we consider a folder lest the get results functions just silently not pick up some scenarios.

I think the regex for an arbitrary repeating sequence of single letters, single numbers, and hyphens would be a bit more complex. I'd have to look into it.